tools/vcli/bw/repl/repl.go

```diff
@@ -175,6 +220,59 @@ func REPL(od storage.Store, input *os.File, rl ReadLiner, chanSize, bulkSize, bu
 			done <- false
 			continue
 		}
+		if strings.HasPrefix(l, "start profiling") {
```

Comment:
For a separate PR/Unralated to your changes:
Consider updating the loop to have a switch instead of several `if strings.HasPrefix` checks. 
Remember: in Go, switch statements are more general than in other languages, so you can have cases checking for `HasPrefix`. 
https://golang.org/doc/effective_go.html#switch
Besides being more readable, it can probably simplify the `done <- false/continue` repetitions. You might want to look at labels to control the flow here, rather than using the channel :)

Reply:
Thank you Tati! It is cool that `switch` can be this general in Go, we will keep this in mind for a future PR as well. =]